projects/Linked List/README.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Good start on this file! This will be the only file we work out of.

Reply:
okay, I'll remove those, so here we do not want the solutions. am I correct?